packages/integration-tests/src/tests/ui/applications/application-test-cases.ts

```diff
@@ -0,0 +1,252 @@
+export type ApplicationCase = {
+  framework: string;
+  name: string;
+  description: string;
+  guideFilename: string;
+  sample?: {
+    repo: string;
+    path: string;
+  };
+  redirectUri?: string;
+  postSignOutRedirectUri?: string;
+};
+
+const nextJs: ApplicationCase = {
+  framework: 'Next.js',
+  name: 'Next.js App',
+  description: 'This is a Next.js app',
+  guideFilename: 'web-next',
```

Comment:
should we read these data from a shared resource. What if the content has been updated?

Reply:
Maybe we can consider copy the metadata ts file from console?

Reply:
Updated.
Per offline discussion, these use cases are generated by templates,so we only need to select one for testing.